editor-components/header/header31/header31.module.scss

```diff
@@ -168,198 +167,185 @@
-                      &:hover {
-                          transform: scale(1.05) !important;
-                          cursor: pointer;
-                      }
                   }
                   
               }
           }
         }
     }
 }
+.container-no-image{
+    background-color: var(--composer-html-background);
+    .max-content {
+        .slider-content {
+            .main-social {
+                color: var(--composer-font-color-primary);
+              .icon-stick {
+                  background-color: var(--composer-font-color-primary);
+              }
+            }
+            .control-nav {
+                color: var(--composer-font-color-primary);
+            }
+            .control-num {
+                color: color-mix(in srgb, var(--composer-html-background), var(--composer-font-color-primary) 5%);
+                -webkit-text-stroke: 1px var(--composer-tertiary-color);
+            }
+            .slider-container {
+                .slider-inner {
+                    .up-text {
+                        .stick {
+                            background-color: var(--composer-font-color-primary);
+                        }
+                        .subtitle {
+                            color: var(--composer-font-color-primary);
+                        }
+                    }
+                    .title {
+                        color: var(--composer-font-color-primary);
+                    }
+                    .description {
+                        color: var(--composer-font-color-primary);
+                    }
+                }
+            }
+        }
+    }
+}
 
-@media only screen and (max-width: 768px) {
-    .container {
-        width: 100%;
+@container (max-width: #{$composer-tablet-width-lg}) {
+    .container ,.container-no-image{
         height: 100%;
-
         .max-content {
-            max-width: var(--composer-content-width);
-            margin: auto;
-
             .slider-parent {
                 .slider {
                     left: unset;
                     width: 100%;
-                }
-
-                .background-right {
-                    position: relative;
-                    object-fit: cover;
-                    left: 23% !important;
-                    width: 80% !important;
+                    .right-slider{
+                        .background-right {
+                            left: 20%;
+                            width: 80%;
+                        }
+                    }
                 }
             }
-
-            .carousel {
-                width: 10% !important;
-            }
-
-            .slider-container{
-                .content{
-                    left: 44px;
+            .slider-content{
+                .slider-container{
+                    .carousel {
+                        width: 10%;
+                        .swiper-wrapper{
+                            .content{
+                                left: 44px;
```

Comment:
gap usage?

Reply:
I changed with 40px and it is also static for container padding